h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -260,6 +266,11 @@ public ModelMetricsListSchemaV3 score(int version, ModelMetricsListSchemaV3 s) {
     if (customMetricFunc == null) {
       customMetricFunc = parms._model._parms._custom_metric_func;
     }
+    // set user given auc type, used for scoring a testing data fe. from h2o.performance function
+    MultinomialAucType at = parms._model._parms._auc_type;
```

Comment:
The auc_type is then used via Model parameters, so that is the reason why I do this here. It is the simplest way, how to propagate a new parameter value. Otherwise, I would have changed the parameters and behavior of many other methods in our Java core.